CONTRIBUTING.md

```diff
@@ -48,8 +45,14 @@ for any of the following plugins, we encourage you to submit it
    fetch from the master repository, not your clone, when running `git fetch`
    et al.)
 
-## Running the examples
 
+## Setting up tools
+
+We use a range of tooling script to do varies of things on CI. (testing, formatting, etc.)
+There are scripts for many common tasks (testing, formatting, etc.) that will likely be useful in preparing a PR.
```

Comment:
You replied Done to the comment about rewriting this, but it doesn't seem to have been applied here.